Formula/x265.rb

```diff
@@ -3,8 +3,8 @@ class X265 < Formula
   homepage "http://x265.org"
 
   stable do
-    url "https://bitbucket.org/multicoreware/x265/downloads/x265_2.6.tar.gz"
-    sha256 "1bf0036415996af841884802161065b9e6be74f5f6808ac04831363e2549cdbf"
+    url "https://bitbucket.org/multicoreware/x265/downloads/x265_2.7.tar.gz"
+    sha256 "d5e75fa62ffe6ed49e691f8eb8ab8c1634ffcc0725dd553c6fdb4d5443b494a2"
 
     # build with nasm when >=2.7 is released
```

Comment:
The `stable do` and `head do` blocks should be removed and nasm should be used for all builds now.

Reply:
I was in the midst of doing just that. Good catch, @Noctem 😄.